### ISSUE TITLE
Can't install Alacritty-v0.2.7_amd64.deb on Ubuntu 16.04

### ISSUE THREAD

**Issue by andreas-scharf** [jwilm/alacritty#2075]
The current release `0.2.7` can't be installed on Ubuntu 16.04. Version `0.2.3` is working.

```shell
 sudo dpkg -i Alacritty-v0.2.7_amd64.deb
[sudo] password for ..:
(Reading database ... 356398 files and directories currently installed.)
Preparing to unpack Alacritty-v0.2.7_amd64.deb ...
Unpacking alacritty (0.2.7) over (0.2.3) ...
dpkg: dependency problems prevent configuration of alacritty:
 alacritty depends on libgcc1 (>= 1:8.2.0); however:
  Version of libgcc1:amd64 on system is 1:6.0.1-0ubuntu1.
 alacritty depends on libfontconfig1 (>= 2.12.6); however:
  Version of libfontconfig1:amd64 on system is 2.11.94-0ubuntu1.1.
 alacritty depends on libexpat1 (>= 2.2.5); however:
  Version of libexpat1:amd64 on system is 2.1.0-7ubuntu0.16.04.3.
 alacritty depends on libc6 (>= 2.27); however:
  Version of libc6:amd64 on system is 2.23-0ubuntu10.
 alacritty depends on zlib1g (>= 1:1.2.11.dfsg); however:
  Version of zlib1g:amd64 on system is 1:1.2.8.dfsg-2ubuntu4.1.
 alacritty depends on libpng16-16 (>= 1.6.34); however:
  Package libpng16-16 is not installed.
 alacritty depends on libfreetype6 (>= 2.8.1); however:
  Version of libfreetype6:amd64 on system is 2.6.1-0.1ubuntu2.3.

dpkg: error processing package alacritty (--install):
 dependency problems - leaving unconfigured
Processing triggers for desktop-file-utils (0.22-1ubuntu5.2) ...
Processing triggers for bamfdaemon (0.5.3~bzr0+16.04.20180209-0ubuntu1) ...
Rebuilding /usr/share/applications/bamf-2.index...
Processing triggers for gnome-menus (3.13.3-6ubuntu3.1) ...
Processing triggers for mime-support (3.59ubuntu1) ...
Errors were encountered while processing:
 alacritty
```

I'm wondering if it's an option to build with an older version than `latest` of the docker image `ubuntu`?
https://github.com/jwilm/alacritty/blob/851e77383ea764b793914696b29b4e6d95632ebb/ci/x86_64/Dockerfile#L1

### System
Ubuntu 16.04 xenial
X11
Linux 4.10.0-42-generic #46~16.04.1-Ubuntu SMP Mon Dec 4 15:57:59 UTC 2017 x86_64 x86_64 x86_64 GNU/Linuxx

**Comment by chrisduerr** [comment]
This is a duplicate of https://github.com/jwilm/alacritty/issues/2038.

Note that the latest docker image has been chosen because it had the best compatability at that point in time. We need to pick which platforms we want to support, if the platforms themselves don't support Alacritty. Unfortunately it's impossible for us to provide precompiled binaries for every linux distro out there, which is why maintainers of most distros are already doing a great job at packaging Alacritty.

While it might be possible to provide a few more precompiled binaries for some linux distros, it seems unlikely that Alacritty will ever be able to supply binaries for every possible combination. I'd advise looking into what your distribution's recommended way of maintaining third party applications is.
